app/src/main/java/com/example/drawingApp/MainViewModel.kt

```diff
@@ -10,4 +11,13 @@ class MainViewModel : ViewModel() {
     @ColorInt
     var primaryColor = Color.BLACK
 
+    val layerNames: MutableList<String> = mutableListOf()
+    var layerCounter = 1
```

Comment:
I don't like the implementation of this saved in a counter. For one, you don't need to store this to be able to calculate it. You can always just count up until you find one you don't have. And what if they delete one later? What if I save this project for reopening later? I have to now store a layerCounter? It's making a dependency on this field that it probably doesn't need

Reply:
your right about it creating the dependency and the future issue with delete, instead it will now search for the first available layer name starting from 1. Removed some redundant code with the removal of model.layerCounter

Reply:
*you're

Reply:
er'uoy